### DISCUSSION TITLE
Pronunciation control/SSML question

### DISCUSSION THREAD

**Discussion by tholonia** [neonbjb/tortoise-tts#795] [Q&A]
I have read that Tortoise supports SSML, but I have not been able to get it to work.  I may be doing ti wrong.  I have a text file with SSML and I use that as input to scripts/tortoise_tts.py.  Does Tortoise support SSML? and if so, how can it that used?

My main issue is that the TTS cannot pronounce "AI." I have tried "A I", "A.I.", "A. I.", "eɪ aɪ," and "AY EYE," but they all sound like the speaker is having a stroke. What is the recommended way to get TTS to enunciate individual characters?

**Comment by Tanoshimi** [comment]
I also would like to be able to use SSML.  I'm trying to:
a) Control pronunciation of names
b) Provide mood or inflection.

I tried using [I am Curious] but it did not work.
